### PR TITLE
Update machy8/webloader-bundle

### PR DESCRIPTION
**Pull request by Machy8**
| Q             | A
| ------------- | ---
| License       | MIT

- Changed directory structure
- Added web_loader.yaml with pre-defined fields

### CODE REVIEW

File: machy8/webloader-bundle/1.1/config/packages/web_loader.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+web_loader:
+    pathPlaceholders:
+    filesCollections:
```
Comment (Nyholm) [inline machy8/webloader-bundle/1.1/config/packages/web_loader.yaml]: Why are these needed? Do they do anything?

Comment (Machy8) [reply machy8/webloader-bundle/1.1/config/packages/web_loader.yaml]: Yes. Path placeholders and files collections are the most used configuration parameters in the webloader. So they are here, prearranged, yet empty.

Comment (stof) [reply machy8/webloader-bundle/1.1/config/packages/web_loader.yaml]: be careful, they are `null` here, not an empty string, due to the way Yaml works. The empty string must be represented as a quoted value. So make sure your bundle handles null values, or update this.

Comment (Machy8) [reply machy8/webloader-bundle/1.1/config/packages/web_loader.yaml]: Weloader have condition check for the null values https://github.com/Machy8/webloader-bundle/blob/master/DependencyInjection/WebLoaderExtension.php#L43

**Comment by Machy8** [comment]
@Nyholm could you please merge this?
